core/src/components/input/input.tsx

```diff
@@ -182,7 +182,8 @@ export class Input implements ComponentInterface {
    * `"stacked"`: The label will appear smaller and above the input regardless even when the input is blurred or has no value.
    * `"fixed"`: The label has the same behavior as `"start"` except it also has a fixed width. Long text will be truncated with ellipses ("...").
```

Comment:
The documentation on this property should include where it is supported (which theme, if it isn't supported on all).

Reply:
Added in https://github.com/ionic-team/ionic-framework/pull/29268/commits/8d4e8f5296966eb538b7c598380af306359e9562.